skype_send_message.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+import bcrypt
+from skpy import Skype
+import time
+parser = argparse.ArgumentParser()
+parent_parser = argparse.ArgumentParser(add_help=False)
+parent_parser.add_argument('--username', action="store")
+ShDo = "" # variable for complete work, ShDo = "shutdown"
+outputfile = "contact"
+print ("Welcome to 'Skype send message'. ")
+username = str(input("Type your username of Skype: "))
```

Comment:
@idumenskyi `input` usage is not secure, please take a look into `raw_input`
https://stackoverflow.com/questions/4915361/whats-the-difference-between-raw-input-and-input-in-python3-x